src/SprykerEco/Zed/UnzerApi/Business/Api/Request/AuthorizableChargeUnzerApiRequest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Request;
+
+use Generated\Shared\Transfer\UnzerApiRequestTransfer;
+use SprykerEco\Zed\UnzerApi\Business\Api\Logger\UnzerApiLoggerInterface;
+use SprykerEco\Zed\UnzerApi\Business\Api\Request\Builder\UnzerApiRequestBuilderInterface;
+use SprykerEco\Zed\UnzerApi\Business\Api\Response\Converter\UnzerApiResponseConverterInterface;
+use SprykerEco\Zed\UnzerApi\Dependency\External\UnzerApiToHttpClientInterface;
+use SprykerEco\Zed\UnzerApi\UnzerApiConfig;
+use Symfony\Component\HttpFoundation\Request;
+
+class AuthorizableChargeUnzerApiRequest extends AbstractUnzerApiRequest
+{
+    /**
+     * @var \SprykerEco\Zed\UnzerApi\UnzerApiConfig
+     */
+    protected UnzerApiConfig $unzerApiConfig;
```

Comment:
you have either to add class properties typehints everywhere or don't use them at all
looking from that 99,9% of code for this epic does not contain typehints, remove them here
same for similar `***Request.php` classes